.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  trailingComma: 'es5',
+  singleQuote: true,
+  jsxSingleQuote: false,
```

Comment:
So we're going for double quote?

Reply:
double in JSX, single elsewhere?

Reply:
Actually, looks like that's the default, so probably not necessary here – will check.